src/extension.ts

```diff
@@ -137,7 +138,14 @@ async function doActivate(_operationId: string, context: vscode.ExtensionContext
             mavenExplorerProvider.refresh();
         })
     );
+    const pomSelector: vscode.DocumentSelector = [{ language: "xml", scheme: "file", pattern: "**/pom.xml" }];
     // completion item provider
-    context.subscriptions.push(vscode.languages.registerCompletionItemProvider([{ language: "xml", scheme: "file", pattern: "**/pom.xml" }], completionProvider, ".", "-"));
+    context.subscriptions.push(vscode.languages.registerCompletionItemProvider(pomSelector, completionProvider, ".", "-"));
     registerCommand(context, "maven.completion.selected", sendInfo, true);
+
+    registerCommand(context, "maven.project.addDependency", async () => {
+        if (vscode.window.activeTextEditor) {
+            await addDependencyHandler(vscode.window.activeTextEditor.document);
```

Comment:
Do we really need to pass the document here? since it can be get from everywhere as long as we import `vscode`

Reply:
Indeed, no need to set a param. And the "add a dependency" command only serves for the current active document, it's another point that we don't need a param. 

The reason I pass the document as the param is, in future there's possibility to "add a dependency" for a project without opening its pom.xml. But for the moment, it doesn't make much sense.

Reply:
param document removed.